pypeit/core/qa.py

```diff
@@ -42,23 +37,30 @@ def set_qa_filename(root, method, det=None, slit=None, prefix=None, out_dir=None
         out_dir = os.getcwd()
     #
     if method == 'slit_trace_qa':
-        outfile = 'QA/PNGs/Slit_Trace_{:s}.png'.format(root)
+#        outfile = 'QA/PNGs/Slit_Trace_{:s}.png'.format(root)
+        outfile = 'PNGs/Slit_Trace_{:s}.png'.format(root)
     elif method == 'slit_profile_qa':
         outfile = 'QA/PNGs/Slit_Profile_{:s}_'.format(root)
     elif method == 'arc_fit_qa':
-        outfile = 'QA/PNGs/Arc_1dfit_{:s}_S{:04d}.png'.format(root, slit)
+#        outfile = 'QA/PNGs/Arc_1dfit_{:s}_S{:04d}.png'.format(root, slit)
+        outfile = 'PNGs/Arc_1dfit_{:s}_S{:04d}.png'.format(root, slit)
```

Comment:
This broke `pypeit_qa_html` script but that has been fixed in
the `x_mage` branch.

Reply:
Sorry, my fault.  The function needs to respect that the user can redefine the `qa_path` now kept in `MasterFrame`.  In editing this, I only edited the names that were created during a nominal execution of `run_pypeit`.

Eventually, I'd like this function to be removed.  I don't like that the names of these files are defined (and buried) in a core method.  I'd rather they be defined by a base class or where the files are actually created.